zulipterminal/model.py

```diff
@@ -576,7 +579,9 @@ def update_message_flag_status(self, event: Event) -> None:
 
     def update_rendered_view(self, msg_id: int) -> None:
         # Update new content in the rendered view
+        pos = 0  # Tracks position
         for msg_w in self.msg_list.log:
+            pos += 1
```

Comment:
Can we use `msg_pos + 1` in the second block rather than keeping track of another variable like this?